clients/js/src/wrap.ts

```diff
@@ -47,22 +50,151 @@ const getOwnerFromAccount = async (
   return accountInfo.value.owner;
 };
 
-export interface ExecuteWrapArgs {
+interface TxBuilderArgs {
+  payer: KeyPairSigner | Address;
+  unwrappedTokenAccount: Address;
+  escrowAccount: Address;
+  wrappedTokenProgram: Address;
+  amount: bigint | number;
+  wrappedMint: Address;
+  wrappedMintAuthority: Address;
+  blockhash: {
+    blockhash: Blockhash;
+    lastValidBlockHeight: bigint;
+  };
+  transferAuthority: Address | TransactionSigner;
+  unwrappedMint: Address;
+  recipientWrappedTokenAccount: Address;
+  unwrappedTokenProgram: Address;
+  multiSigners?: TransactionSigner[];
+}
+
+// Used to collect signatures
+export const multisigOfflineSignWrap = async (args: Required<TxBuilderArgs>) => {
+  return buildWrapTransaction(args);
+};
```

Comment:
I think it'd be better to define a separate type with `multiSigners` being non-null, in case another optional field gets added in the future that shouldn't be required by this function

That could be something like:

```
interface TxBuilderArgsWithMultiSigners extends Omit<TxBuilderArgs, 'multiSigners'> {
  multiSigners: TransactionSigner[];
}
```

I'd also use the `function` form here:

```
export aysnc function multisigOfflineSignWrap(args: TxBuilderArgsWithMultiSigners): Promise<FullySignedTransaction> {
  return buildWrapTransaction(args)
}
```